build-config/resources/Ballerina.toml

```
@@ -2,7 +2,7 @@
 distribution = "2201.10.0"
 org = "ballerinax"
 name = "hubspot.marketing.forms"
-version = "@toml.version@"
+version = "0.1.0"
```

Don't change the version in this file.

```suggestion
version = "@toml.version@"
```